scripts/scada_parquet_transformer.py

```diff
@@ -0,0 +1,36 @@
+from time import sleep
+from json import dumps,loads
+from kafka import KafkaProducer
+from kafka import KafkaConsumer
+import sys
+import datetime
+import pyarrow as pa
+import pyarrow.parquet as pq
+import json
+
+broker=str(sys.argv[1])
+consume_topic=str(sys.argv[2])
+produce_topic=str(sys.argv[3])
+
+consumer = KafkaConsumer(
+     consume_topic,
+     bootstrap_servers=[broker],
+     auto_offset_reset='earliest')
+
+producer = KafkaProducer(bootstrap_servers=[broker],
+                         value_serializer=lambda x: 
+                         dumps(x).encode('utf-8'),
+                         client_id="scada-json-producer")
+
+for message in consumer:
+    # Read the kafka record as raw bytes and transform to a pandas dataframe 
+    reader = pa.BufferReader(message.value)
+    table = pq.read_table(reader)
+    df = table.to_pandas()
```

Comment:
```suggestion
    df = pq.read_table(pa.BufferReader(message.value)).to_pandas()
```
By one-lineing this assignment we will get a smaller memory usage. Since the messages coming in are entire tables, it does not make sense to keep it in memmory both as a `table` and `df`.

Reply:
I always prefer readability over memory usage in applications where memory is not a critical factor.